app/components/PublishOverlay.tsx

```diff
@@ -93,6 +94,18 @@ export function PublishOverlay(props: {
         <p>
           Thanks for constructing a puzzle! <Emoji symbol="😎" />
         </p>
+        {(!(props.toPublish.pv === undefined || props.toPublish.pv === false) && (props.toPublish.pvu === undefined)) ? (
+          <p>
+            This puzzle will be posted as private.
+          </p>
+          ) : null
+        }
+        {(props.toPublish.pvu && (props.toPublish.pvu.toMillis() > Date.now()))? (
```

Comment:
Perhaps there's a more elegant way to handle this?

Reply:
Yeah the pvu thing is a bit weird but using it as a flag for "When was/is this puzzle public if it was/is ever public?" allows us to efficiently perform the queries for any sort of time ordered list of public puzzles (e.g. the homepage or a constructor blog).

I don't think the line above is any clunkier than any of the rest of the code dealing w/ pvu 😂